Source/itemdat.h

```diff
@@ -13,7 +13,7 @@
 namespace devilution {
 
 /** @todo add missing values and apply */
-enum _item_indexes : uint16_t {
+enum _item_indexes : int16_t {

```

Comment:
`_item_indexes` type variables are occasionally used as array indexes which makes me think it'd be best to keep this as an unsigned type, but at the same time this type is often used interchangeably with `int32_t` in functions like `IsItemAvailable()`. I've logged #2300 to cover that investigation.